### PR TITLE
SIDE-1914: Add User Aptitude

### PR DESCRIPTION
**Pull request by DanielWagersOlive**
- Test in Self Test Loop just checks for a populated token
- Some formatting changes were included

### CODE REVIEW

File: ldk/javascript/src/user/index.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * The User aptitude gives access to Olive Helps user related information
+ */
+import { promisify } from "../promisify";
+
+export interface User {
+    /**
+     * Returns a JWT with the current username in the subject field
```
Comment (cshields-oliveai) [inline ldk/javascript/src/user/index.ts:8]: You'll want to use the `@returns` annotation here so its parsed by typedoc I do believe.

File: ldk/javascript/src/user/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
```
Comment (cshields-oliveai) [inline ldk/javascript/src/user/index.ts:1]: We added basic unit tests for all of the other aptitudes. I'd vote to keep this aptitude consistent (even though they are just asserting that calls are being made to our `oliveHelps` reference).

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/src/user/index.ts (hunk 2 above)
Comment (DanielWagersOlive) [reply ldk/javascript/src/user/index.ts:1]: Not a fan.  The tests are extremely low value (imo), don't document anything, and probably add more confusion and build time.  But I'll do it if the team values consistency.

File: ldk/javascript/src/user/index.ts (hunk 1 above)
Comment (DanielWagersOlive) [reply ldk/javascript/src/user/index.ts:8]: Thanks.  Will do.

Comment (DanielWagersOlive) [reply ldk/javascript/src/user/index.ts:8]: Done.

**Review by BrettMugglinOlive — APPROVED** [review]

**Pull request merged**